lib/v2/screens/sign_up/viewmodels/signup_event.dart

```diff
@@ -26,8 +27,12 @@ class UnpackScannedLink extends SignupEvent {
   String toString() => 'UnpackScannedLink event { scannedLink: $scannedLink }';
 }
 
-class NavigateToDisplayName extends SignupEvent {}
+class NavigateToDisplayName extends SignupEvent {
+  @override
+  String toString() => 'NavigateToDisplayName event';
```

Comment:
Check your prints like  `NavigateToDisplayName event` to just `NavigateToDisplayName` we already know are events when we see the console logs.